### ISSUE TITLE
Investigate and remediate certificate renewal failure

### ISSUE THREAD

**Issue by jcscottiii** [web-platform-tests/wpt.live#57]
@foolip reported that the certificate for the wpt.live was getting to close to expiration.

---
# Problem
![Screenshot 2022-07-07 2 59 58 PM](https://user-images.githubusercontent.com/7788930/178033308-a9485150-b66d-4dd2-874f-444e21ebd2f8.png)

---
# Diagnosis
## Screenshot 1 - CPU Pegged
![Screenshot 2022-07-07 2 51 34 PM](https://user-images.githubusercontent.com/7788930/178032088-a1626cd2-1476-40bb-b402-929a9c980103.png)

## Screenshot 2 - No space left on the device
![Screenshot 2022-07-07 2 58 54 PM](https://user-images.githubusercontent.com/7788930/178033276-101b3e08-9c94-453a-b1ad-57ac96c02b10.png)

## Screenshot 3 - Bucket has not been touched in awhile
![Screenshot 2022-07-07 3 00 48 PM (1)](https://user-images.githubusercontent.com/7788930/178033846-2cf67d04-5d89-47a3-ad14-95ec075de591.png)

## Screenshot 4 - Logs do not indicate anything wrong
![Screenshot 2022-07-07 2 57 54 PM](https://user-images.githubusercontent.com/7788930/178035251-65f17e57-b385-486b-ba1e-cc3ee1849651.png)

## Screenshot 5 - Unable to log into the instance
![image](https://user-images.githubusercontent.com/7788930/178034953-1cb73874-21f5-4eb0-8fad-9e82d3ac1f0d.png)

## Summary
CPU was pegged. There has to be some process that was gotten out of control since the cert renewal is only a cert bot script that runs once a day. Also something is causing the server to use up all the space. As a result of all of this, I could not log in to do further diagnosis.  Need to restart/recreate the instance

---

# Remediation Steps

- [x] Restart the instance
  - Upon restarting, ran into a problem where us-central1 ran out of the instance type f1-micro
  - ![image](https://user-images.githubusercontent.com/7788930/178036545-edb1c5ef-53c7-430b-9fc9-edca5cbe8fff.png)
  - I attempted to recreate the instance in all of the zones in us-central1 and got the same error.
  - This error is still happening across two days
- [x] Find a new instance type that will successfully provision: e2.micro
- [x] Discuss the increase in billing with @past 
- [x] Verify that the certificate is indeed updated
  - ![image](https://user-images.githubusercontent.com/7788930/178038039-97baeadf-d830-434e-9591-4594e77fc06d.png)
- [x] ~Create a PR with the new instance type to the terraform to reflect the new state of infrastructure~ No longer needed

Will close this issue after finishing these steps

# Other recommendations
If we want to save money, move the cert-renewal to Cloud Run instance that starts up, runs, and terminates upon a cron schedule. This will remove the need to have an instance constantly on.

cc: @DanielRyanSmith

**Comment by past** [comment]
There is some investigation of an earlier instance of the disk running out of space in #35 that could be related.

**Comment by jcscottiii** [comment]
There was another failure to renew the cert recently. I did not check the serial console for the out of space  error prior to restarting. But I suspect that it is the same problem. I have added some alerts If this happens again, we should migrate to the Cloud Run instance in the "Other Recommendations" section above. It would be easier to do that change instead of chasing down what's causing the instance to eventually run out of disk space.
